src/datasources.js

```diff
@@ -306,7 +314,11 @@ define([
             };
         }
         function getTree(item, id, path, info) {
-            var tree = {name: item.name || id, recursive: false},
+            var tree = {
+                    name: item.name || id,
+                    description: item.description || '',
+                    recursive: false
+            },
```

Comment:
indeed. Not sure how it got that way, but I'll fix it the next time I visit that code. Going to merge now on account of how long it takes to run tests these days.